### DISCUSSION TITLE
Colab error "cohere" "openai" not installed

### DISCUSSION THREAD

**Discussion by emuchemi** [openai/whisper#2030] [Q&A]
While installing on Google Colab, it completed installation and transcribing can function, but there were these errors that popped up during installation. How do you think I should handle it? and can it impede the quality of the transcription?

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
llmx 0.0.15a0 requires cohere, which is not installed.
llmx 0.0.15a0 requires openai, which is not installed.

**Comment by glangford** [comment]
> How do you think I should handle it? and can it impede the quality of the transcription?

It is irrelevant to the operation of whisper.
